NLibrary/Classes/Service Files/Mocks/FirebaseServiceMock.swift

```diff
@@ -9,6 +9,10 @@
 import Foundation
 
 public class FirebaseServiceMock: NSObject, FirebaseServiceProtocol {
+    public required override init() {
+        
+    }
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5382838)